mesonbuild/modules/gnome.py

```diff
@@ -886,6 +885,107 @@ def mkenums(self, state, args, kwargs):
         else:
             return ModuleReturnValue(targets, targets)
 
+    def mkenums_simple(self, state, args, kwargs):
+        hdr_filename = args[0] + '.h'
+        body_filename = args[0] + '.c'
+
+        # not really needed, just for sanity checking
+        forbidden_kwargs = ['c_template', 'h_template', 'eprod', 'fhead',
```

Comment:
Yes, but a whitelist will also protect against typos of kwargs that we do accept, that's all.